src/std/text.ab

```diff
@@ -1,62 +1,73 @@
+/// Replace the content just once
 pub fun replace_once(source, pattern, replacement) {
     return unsafe $echo "\$\{source/{pattern}/{replacement}}"$
 }
 
+// Replace the content multiple times
 pub fun replace(source, pattern, replacement) {
     return unsafe $echo "\$\{source//{pattern}/{replacement}}"$
 }
 
+/// Replace with regex using `sed`
 pub fun replace_regex(source: Text, pattern: Text, replacement: Text): Text {
     return unsafe $echo "{source}" | sed -e "s/{pattern}/{replacement}/g"$
 }
 
-
+/// Split the string and return an array
 pub fun split(text: Text, delimiter: Text): [Text] {
     let result = [Text]
     unsafe $IFS="{delimiter}" read -rd '' -a {nameof result} < <(printf %s "\${nameof text}")$
     return result
 }
 
+/// Split a string with newlines
 pub fun lines(text: Text): [Text] {
-    let result = [Text]
-    unsafe $IFS=\$'\n' read -rd '' -a {nameof result} <<<"\${nameof text}"$
-    return result
+    return split(text, "\n")
 }
 
+/// Split by space separator
 pub fun words(text: Text): [Text] {
```

Comment:
It should return an array instead of a string?

Reply:
an array of all words